app/src/components/AnalysisPane.js

```diff
@@ -1,147 +1,194 @@
-      <h4>{result.Text}</h4>
-
-      <dl>
-        {attrs.map(([ key, value ]) => (
-          <>
-            <dt>{key}</dt>
-            <dd>{value}</dd>
-          </>
+    <Flex width='100%' alignItems='center'>
+      <Select
+        mr={2}
+        border='1px solid'
+        borderColor={concepts[0].Score < 0.5 ? 'red' : 'grey'}
+        borderRadius='0'
+        bg='white'
+        _hover={{ borderColor: concepts[0].Score < 0.5 ? 'red' : 'grey', boxShadow: 'none' }}
+      >
+        {concepts.map((concept) => (
+          <option key={concept.Code} value={concept.Code}>
+            {result.Text} {concept.Code ? ' | ' : null} {concept.Code} {concept.Description ? ' | ' : null}{' '}
+            {concept.Description} &nbsp;&nbsp; {(concept.Score * 100).toPrecision(4)}%
+          </option>
         ))}
-      </dl>
-    </div>
-  )
-}
+      </Select>
 
-function ResultTable({
-  results,
-  category,
-  onToggleItem,
-  excludedItems
-}) {
-  const filteredResults = useMemo(() => results.filter(r => r.Category === category), [ results, category ]);
-  const [ open, setOpen ] = useState(false);
-
-
-  return <div className={s.resultTable}>
-    <h3 className={open ? s.expanded : s.contracted} onClick={() => setOpen(x => !x)}>{displayNames[category]}</h3>
-
-    <HeightSlider>
-      {open ?
-        <div>
-          {filteredResults.map((r, i) => (
-            <ResultRow
-              result={r}
-              key={r.id}
-              onToggleItem={onToggleItem}
-              excludedItems={excludedItems}
-            />
-          ))}
-        </div>
-      : null }
-    </HeightSlider>
-  </div>
+      {closeIcon}
+    </Flex>
+  );
 }
 
+function ResultTable({ results, category, onToggleItem, excludedItems, onResultDelete, onResultAdd }) {
+  const filteredResults = useMemo(() => results.filter((r) => r.Category === category), [results, category]);
+  const [inputValue, setInputValue] = React.useState('');
+  const handleInputChange = (event) => setInputValue(event.target.value);
+
+  const addIcon = (
+    <IconButton
+      aria-label='Add'
+      icon={<AddIcon />}
+      onClick={() => [inputValue.trim() !== '' ? onResultAdd(inputValue, category) : null, setInputValue('')]}
+      size='sm'
+      borderRadius='0'
+      color='grey'
+    />
+  );
 
-
+  return (
+    <Box className={s.resultTable} mb={4}>
+      <Box as='h1' mb={4} textAlign='left' fontWeight='bold' fontSize='1.2rem'>
+        {displayNames[category]}
+      </Box>
+
+      <VStack spacing={2}>
+        <Flex width='100%' alignItems='center'>
+          <Input
+            mr={2}
+            mb={4}
+            border='1px solid'
+            borderColor='grey'
+            borderRadius='0'
+            bg='white'
+            placeholder={'Add ' + displayNames[category]}
+            value={inputValue}
+            onChange={handleInputChange}
+          ></Input>
```

Comment:
Components/elements not receiving a `children` prop should be self-closing in JSX.